src/auction.js

```diff
@@ -462,9 +462,12 @@ export function auctionCallbacks(auctionDone, auctionInstance, {index = auctionM
     handleBidResponse(adUnitCode, bid, (done) => {
       let bidResponse = getPreparedBidForAuction(bid);
 
-      if (bidResponse.mediaType === 'video') {
+      if (bidResponse.mediaType === VIDEO) {
         tryAddVideoBid(auctionInstance, bidResponse, done);
       } else {
+        if (bidResponse.mediaType === NATIVE) {
```

Comment:
`if(FEATURES.NATIVE && bidResponse.mediaType === NATIVE)`. Without the additional check the build would pull native-specific code even when the user asks to remove it.